### ISSUE TITLE
Set token expiration and force_revokable from Artifactory 7.50.3

### ISSUE THREAD

**Issue by TJM** [jfrog/artifactory-secrets-plugin#40]
https://www.jfrog.com/confluence/display/JFROG/JFrog+Platform+REST+API#JFrogPlatformRESTAPI-CreateToken

> From Artifactory 7.50.3, the "force revocable" flag in the tokens has been removed as a default setting and is now a Boolean parameter called force_revocable in the Create Token REST API. When this parameter is set to true, we will add the force_revocable flag to the token's extension.

I *think* this will allow us to set the expiration date on the tokens themselves too, while still allowing them to be revokable (without messing with the config)... This will help prevent the "orphaned tokens" we are seeing periodically.

**Comment by alexhung** [comment]
Yes, as long as the expiration date is translated to number of seconds and that value does not exceed the system's max expiry.

**Comment by TJM** [comment]
I wonder if those values are queryable? By default, max-expiration appears to be 0 (unlimited) and `0` is also what the code is using, so if they have a max expiration set, I'd bet this code would fail. (testing that)

**Comment by alexhung** [comment]
> I wonder if those values are queryable?

Not that I'm aware of through APIs.
